core/templates/components/on-screen-keyboard/on-screen-keyboard.component.ts

```diff
@@ -44,13 +44,15 @@ export class OnScreenKeyboardComponent {
   guppyInstance!: Guppy;
   interactionType!: string;
   customLetters!: string[];
-  functionsTab: string = constants.OSK_FUNCTIONS_TAB;
-  lettersTab: string = constants.OSK_LETTERS_TAB;
-  mainTab: string = constants.OSK_MAIN_TAB;
+  functionsTab: string = AppConstants.OSK_FUNCTIONS_TAB;
+  lettersTab: string = AppConstants.OSK_LETTERS_TAB;
+  mainTab: string = AppConstants.OSK_MAIN_TAB;
   greekSymbols: string[] = Object.values(
-    constants.GREEK_LETTER_NAMES_TO_SYMBOLS);
+    AppConstants.GREEK_LETTER_NAMES_TO_SYMBOLS);
+
```

Comment:
I think this empty line can be removed.

Reply:
linter checks cannot pass without this line